jest/private/jest_test.bzl

```diff
@@ -89,7 +115,7 @@ def _impl(ctx):
         # https://jestjs.io/docs/cli#--configpath. The path to a Jest config file specifying how to
         # find and execute tests.
         "--config",
-        generated_config.short_path,
+        paths.join(inv_chdir, generated_config.short_path),
```

Comment:
I think we can just go with the simple case for now and assume there will be no `.` or `..` in the path. If we decide to support more complex paths we should do it across all rulesets.

See https://docs.aspect.build/rules/aspect_rules_js/docs/js_binary/#chdir for the `..` snippet, or webpack has [a simple one](https://github.com/aspect-build/rules_webpack/blob/04b6a2714bfff1b55dc166048b32e59afd0e9268/webpack/private/webpack_devserver.bzl#L45-L47) you can copy.

Reply:
Thanks for the feedback, I updated the commit to reflect how this is done in webpack

Reply:
I think you also need to remove the test containing the `.` and `..`s now though, see the test errors.